src/ui/classic/xcbinputwindow.cpp

```diff
@@ -127,13 +127,12 @@ void XCBInputWindow::updateDPI(InputContext *inputContext) {
 }
 
 void XCBInputWindow::update(InputContext *inputContext) {
-    if (!wid_) {
+    if (!wid_ || !inputContext) {
```

Comment:
inputContext is allowed to be null and it will hide the window.

Reply:
done